t/fuzz_mmdb.c

```diff
@@ -0,0 +1,36 @@
+#include "maxminddb-compat-util.h"
+#include "maxminddb.h"
+#include <unistd.h>
+
+#define kMinInputLength 2
+#define kMaxInputLength 4048
+
+extern int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
+
+int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    int     status;
+    FILE    *fp;
+    MMDB_s  mmdb;
+    char    filename[256];
+
+    if (size < kMinInputLength || size > kMaxInputLength)
+        return 0;
+
+    sprintf(filename, "/tmp/libfuzzer.%d", getpid());
+
+    fp = fopen(filename, "wb");
+    if (!fp)
+        return 0;
```

Comment:
Should this or the above return something other than 0?

Reply:
> Should this or the above return something other than 0? 

No, if the program returns anything else other than `return 0;`,  the coverage won't be counted.
I get it; we could use `return -1;` or `return 1;`, but I would say let's keep it `return 0;` for consistency.

Souce [LibFuzzer](https://llvm.org/docs/LibFuzzer.html#rejecting-unwanted-inputs)

Reply:
Would we want it to be counted though? It's an error, right?

Reply:
> It's an error, right?

It is an error. I think there are bigger problems in your system if you see these errors.
We would be running in a controlled environment, so it won't be a problem.